### PR TITLE
docs: fix theming docs misspells pink-bluegrey.css as pink-bluegray.css

### PR DESCRIPTION
**Pull request by AlirezaEbrahimkhani**
rename pink-bluegray.css to pink-bluegrey.css

resolves angular#24378

### CODE REVIEW

File: guides/theming.md [hunk 1]
```diff
@@ -286,7 +286,7 @@ $light-theme: mat.define-light-theme((
 
 // Define a dark theme
 $dark-primary: mat.define-palette(mat.$pink-palette);
-$dark-accent: mat.define-palette(mat.$blue-gray-palette);
+$dark-accent: mat.define-palette(mat.$blue-grey-palette);
```
[the remark this answers is not in the document]
Comment (AlirezaEbrahimkhani) [reply guides/theming.md:289]: Yes you're right.
so what should i do now? close the PR ?

[the remark this answers is not in the document]
Comment (crisbeto) [reply guides/theming.md:289]: I don't think that we need to make any changes given that both spellings are valid.

[the remark this answers is not in the document]
Comment (AlirezaEbrahimkhani) [reply guides/theming.md:289]: so i think i can close this PR
Thanks for your feedback and review :pray: :heart:

**Comment by AlirezaEbrahimkhani** [comment]
This PR closed based on [Kristiyan Kostadinov](https://github.com/crisbeto)'s review comment.

**Pull request closed without merging**